### DISCUSSION TITLE
Github not working

### DISCUSSION THREAD

**Discussion by yashnaresh** [community/community#137214] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Github for my family member is not opening with correct credentials and when trying to do forget password, no email is coming. Can anyone suggest what can be the issue?

**Comment by theshawnv** [comment]
hello @yashnaresh, whatsup,
These issues kind of came from various reasons, it could be your IP or region and it could be you connect some proxy or DNS on your local machine.

**Comment by mr0-nerd** [comment]
Same thing is happening with me today.
I cannot access my GitHub account and when I am trying to click on reset password link then no link is comming to my mail.
Also when I am trying to open my GitHub profile as guest then it's showing 404 . I have tried to open it from various browsers but same 404. 
There is no mail on my account regarding flagging or hiding my account.
Everything was alright last night 😭
Please someone help me

**Reply by theshawnv** [reply]
did you Check Your Email and Spam Folder? or did you Check for Account Status Notifications?

**Reply by mr0-nerd** [reply]
No mail received regarding reset password or regarding account status

**Comment by yashnaresh** [comment]
Yes, there is no mail from github, tried multiple times

**Reply by mr0-nerd** [reply]
Did your issue got resolved yet??

**Comment by yashnaresh** [comment]
not yet
